test/e2e-helm/e2e_helm_olm_test.go

```diff
@@ -81,6 +70,12 @@ var _ = Describe("Integrating Helm Projects with OLM", func() {
 				"--timeout", "4m")
 			_, err = tc.Run(runPkgManCmd)
 			Expect(err).NotTo(HaveOccurred())
+
+			By("destroying the deployed package manifests-formatted operator")
+			cleanupPkgManCmd := exec.Command(tc.BinaryName, "cleanup", projectName,
+				"--timeout", "4m")
```

Comment:
Follow-up: helm's e2e tests should be using the test namespace, and `run`/`cleanup` should have `--namespace=<tc.Kubectl.Namespace>` in their command args.